travis_script_1.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+echo heloo
```

Comment:
to be used later when we add more features to webapp without adding more variables but sending just a script to be executed inside chroot for making changes